terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,36 @@
+aws_bucket = "development-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-development-terraform-state-london"
+environment = "cidev"
+deploy_to = "development"
+state_prefix = "env:/development"
+aws_profile = "development-eu-west-2"
+kms_alias = "alias/development-eu-west-2/environment-services-kms"
+
+# service configs
+# Commented params moved to vault service secrets
+# cache_server 
+# oauth2_redirect_uri 
+# oauth2_auth_uri 
+# account_url 
+log_level = "debug"
+chs_url = "https://cidev.aws.chdev.org"
+piwik_url = "https://matomo.platform.aws.chdev.org/"
+piwik_site_id = "7"
+cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
+chips_presenter_auth_url = "http://enablement-presenter-api:8080/presenterauth"
+chs_company_profile_api_local_url = "https://api.cidev.aws.chdev.org"
+cache_db = "0"
+dissolutions_api_url = "https://api.cidev.aws.chdev.org"
```

Comment:
Aware we've discussed this already but this is a global var in the mesos setup, worth replicating?

Reply:
Covered already in conversation on node-18-upgrade

Reply:
Yup, we're discussing setting up global vars lookup